src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/SupplementaryEvidenceCreatorTest.java

```diff
@@ -0,0 +1,131 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.model.ccd.SupplementaryEvidence;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.model.ccd.mappers.SupplementaryEvidenceMapper;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Envelope;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDataContent;
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.BDDMockito.willThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData.AUTH_DETAILS;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData.SERVICE_TOKEN;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData.USER_DETAILS;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData.USER_TOKEN;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SupplementaryEvidenceCreatorTest {
+
+    private static final String CASE_TYPE_ID = "Bulk_Scanned";
+    private static final String EVENT_TYPE_ID = "attachScannedDocs";
+
+    @Mock
+    private CcdAuthenticatorFactory authenticatorFactory;
+
+    @Mock
+    private CoreCaseDataApi coreCaseDataApi;
+
+    private SupplementaryEvidenceCreator creator;
+
+    @Before
+    public void setUp() {
+        given(authenticatorFactory.createForJurisdiction(any())).willReturn(AUTH_DETAILS);
+
+        creator = new SupplementaryEvidenceCreator(authenticatorFactory, coreCaseDataApi);
+    }
+
+    @Test
+    public void createSupplementaryEvidence_starts_and_submits_event() {
```

Comment:
what benefit does this give us?
The verifications seem to be around that CCD calls happened, to we care how they happy?
What we're interested in is did the update persist in CCD, not the api calls needed to get it there?

Reply:
True, although as an external client is provided and we assume it's reliable, checking if it's used correctly (i.e. called with the right parameters) adds, in my opinion, some value. It doesn't check if the configuration is correct, of course, and this test doesn't replace integration tests.

Reply:
Sure I guess in our case its worth checking we are using the right jurisdiction and user token